test/svt/src/main/java/quarks/test/svt/apps/FleetManagementAnalyticsClientApplication.java

```diff
@@ -0,0 +1,66 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+ */
+package quarks.test.svt.apps;
+
+import quarks.test.svt.apps.iotf.AbstractIotfApplication;
+import quarks.test.svt.utils.sensor.gps.GpsSensor;
+import quarks.topology.Topology;
+
+/**
+ * A Global Positional System and On-Board Dashboard application to perform analytics 
+ * defined in {@link GpsAnalyticsApplication} and {@link ObdAnalyticsApplication}.  
+ * <p>
+ * The Quarks console URL is written to the console and to file consoleUrl.txt.
+ * <p> 
+ * The Watson IotF URL is written to the console and to file iotfUrl.txt
+ * 
+ * <p>
+ * Argument: specify pathname to application properties file. If running in Eclipse, you can  
+ *  	specify GpsObdAnalyticsApplication.properties.
+ */
+public class FleetManagementAnalyticsClientApplication extends AbstractIotfApplication {
+
```

Comment:
semi-nit: the rest of the Quarks code avoids the use of hard-tabs (or tries to :-)